src/app/(post-login)/submission/layout.tsx

```diff
@@ -32,7 +32,7 @@ export default function AppLayout({ children }: { children: ReactNode }) {
 		<div
 			css={css`
 				display: grid;
-				grid-template-columns: ${isSidebarActive ? '248px' : '40px'} 1fr;
+				grid-template-columns: ${isSidebarActive ? '248px 86vw' : '40px 98vw'};
```

Comment:
Insure Submission content fits in view window

Reply:
I haven't seen mixing of px and view port units before - interesting!
keep in mind `vw` and `vm` are percents and `px` are absolute

Reply:
@ciaranschutte the unit mix makes sense if you see height in relation to a "constant" font size (8'd have opted for rem) vs viewport width depending on browser being resized

Reply:
wouldn't this potentially result in overflow/underflow though? 
if 248px is not 14% (100 - 86) of the view port width  .. that's one col of fixed size, and another col of fixed size that is 86% of viewport. 

are you talking about that idea of setting a base font size and setting everything, including layout dimensions, relative to it?

Reply:
oh, I was strictly focused on the unit mix... the actual values, I have no clue about 🤷‍♂️

Reply:
@demariadaniel  not sure where those numbers came from but could we change this to, otherwise the page gets a little overflowing:
```
 grid-template-columns: ${isSidebarActive
          ? "248px calc(100vw - 248px)"
          : "40px calc(100vw - 40px)"};
```

Reply:
Oh. I see what you meant!
Yeah, this suggesrion certainly seems more robust. 

I'd take it a step further by abstracting those constants into reusable variables, that also go into the sidebar itself (so if the number changes at some point, it does it in both places at once)

Reply:
<insert "100" emoji>

Reply:
Great catch @ciaranschutte this did indeed overflow + calc is exactly what I was looking for

For context, I added the `vw` changes to contain the table within the page:

<img width="1284" alt="Screenshot 2023-10-27 at 12 11 44 PM" src="https://github.com/icgc-argo/rdpc-ui/assets/15621876/9fade8f6-873b-461c-9d14-f1d5e5c84189">
<img width="1059" alt="Screenshot 2023-10-27 at 12 11 34 PM" src="https://github.com/icgc-argo/rdpc-ui/assets/15621876/c9d83f0a-122a-4b9b-b5e0-645148e7055c">

Reply:
Using vw the table fits on the page:
<img width="1290" alt="Screenshot 2023-10-27 at 12 10 42 PM" src="https://github.com/icgc-argo/rdpc-ui/assets/15621876/1f8825f9-ff15-4763-bcb4-dc19c0a490cd">